modules/consensus/accept_bench_test.go

```diff
@@ -0,0 +1,44 @@
+package consensus
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+// benchmarkEmptyBlocks is a benchmark that mines many blocks, and
+// measures how long it takes to add them to the consensusset
+func benchmarkEmptyBlocks(bt *testing.B) error {
+	bt.StopTimer()
+
+	// The test dir will be reset each time the benchmark
+	// is done.
+	cst, err := createConsensusSetTester("BenchmarkEmptyBlocks")
+	if err != nil {
+		return errors.New("Error creating tester: " + err.Error())
+	}
+
+	for j := 0; j < bt.N; j++ {
+		b, _ := cst.miner.FindBlock()
+		bt.StartTimer()
+		err := cst.cs.AcceptBlock(b)
+		if err != nil {
+			errstr := fmt.Sprintf("Error accepting %d: %s", j, err.Error())
+			return errors.New(errstr)
+		}
+		cst.csUpdateWait()
+		bt.StopTimer()
```

Comment:
Actually, because of changes I'm planning to make to the subscription model, it might be better not to use a full consensus tester, which I believe also contains a wallet and a transaction pool.

I had some ideas for other ways to approach this benchmark, but our debugging tools and DoS protections actually make benchmarking the consensus package difficult. It'd be easier to start with the lower level functions like StandaloneValid, applyMaturedSiacoinOutputs, and such.

One way around the debugging tools would be to have a `make bench` which runs with debugging disabled.